ipmininet/router/config/zebra.py

```diff
@@ -11,6 +12,15 @@
 PERMIT = 'permit'
 
 
+def get_family(prefix: Union[IPv4Network, IPv6Network]) -> Optional[str]:
```

Comment:
The prefix can also be a string to convert to a prefix I think
`Union[str, IPv4Network, IPv6Network]`

The check would be `ip_network(str(prefix)).version == 4`, like ip_statement actually

Reply:
Changed: dca805acddbe19863aea03c4ed488785ca0b650e